crates/load-test/src/main.rs

```diff
@@ -0,0 +1,450 @@
+//! Load test for pathfinder JSON-RPC endpoints.
+//!
+//! This program expects a fully synced mainnet pathfinder node, since it contains
+//! references to transactions and contract hashes on mainnet.
```

Comment:
Maybe specify the actual (rough) block number required? That way we know for the far future, when mainnet is rather large.

Reply:
Latest block on main is `2003`.